rasa/core/utils.py

```diff
@@ -477,7 +477,7 @@ def replace_floats_with_decimals(obj: Union[List, Dict]) -> Any:
             obj[j] = replace_floats_with_decimals(obj[j])
         return obj
     elif isinstance(obj, float):
-        return Decimal(obj)
+        return round(Decimal(obj), 9)
```

Comment:
can we make the number of decimals a parameter of the function? `9` could be the default value